docker/mainnet/dash/dash.mainnet.conf

```diff
@@ -0,0 +1,11 @@
+port=10333
+rpcport=10332
+
+rpcallowip=172.100.2.100
+
+txindex=1
```

Comment:
Why do we need txindex in dash?

Reply:
Not sure, but it was in testnet and simnet in previous installations